pallets/connectors/src/message.rs

```diff
@@ -35,34 +35,93 @@ where
 	Rate: Encode + Decode,
 {
 	Invalid,
+	AddCurrency {
+		currency: u128,
+		evm_address: [u8; 20],
+	},
 	AddPool {
 		pool_id: PoolId,
 	},
+	AllowPoolCurrency {
+		currency: u128,
+		pool_id: PoolId,
+	},
```

Comment:
In the solidity implementation as well as in the spec, the order of these parameters was reversed. See discussion here: https://github.com/centrifuge/connectors/pull/59#discussion_r1174540624

Reply:
The spec is updated ☝️